src/components/timeline/TimelineHistogram.svelte

```diff
@@ -360,15 +392,34 @@
   />
 
   <div
-    class="histogram blue"
+    class="histogram"
     style={`height: ${
       constraintViolationsToRender.length ? histogramHeight * 1.5 : histogramHeight * 2
     }px; border-bottom: ${constraintViolationsToRender.length < 1 ? '1px solid var(--st-gray-20)' : 'none'}`}
   >
-    {#each activityHistValues as bin, index (index)}
-      <div class="bin" style={`height: ${(bin / activityHistMax) * 100}%;`} />
+    {#each aggregateHistogram as binPair, index (index)}
+      <!-- TODO: MAKE THE MAX AND BINNING GLOBAL INSTEAD OF SPECIFIC TO EVENT AND ACTIVITY-->
+      {@const activityHeight = binPair[0] / aggregateMax}
+      {@const externalEventHeight = binPair[1] / aggregateMax}
+      {#if binPair[0] && !binPair[1]}
+        <div class="bin-item blue" style={`height: ${activityHeight * 100}%;`} />
+      {:else if binPair[1] && !binPair[0]}
+        <div class="bin-item orange" style={`height: ${externalEventHeight * 100}%;`} />
+      {:else if binPair[0] && binPair[1]}
+        <div class="bin-item" style="height: 100%;">
+          <div
+            class="bin-item blue"
+            style={`height: ${(1 - (activityHeight + externalEventHeight)) * 100}%; width: 0%`}
+          />
+          <div style={`height: ${activityHeight * 100}%; background-color: rgba(47, 128, 237, 1);`} />
+          <div style={`height: ${externalEventHeight * 100}%; background-color: rgba(237, 158, 47);`} />
+        </div>
+      {:else}
+        <div class="bin-item blue" style={`height: ${0}%;`} />
```

Comment:
Can the static height be moved to a class definition?

Reply:
Does this affect the layout with a `height` of `0`?

Reply:
Yes, as it has a non-zero width. Without this, there are no gaps between each colored bin; they just take up the whole space. These zero-height rows are very necessary!

Reply:
Can we rename this class to `gap` instead of `invisible` and rather than doing a 0 height, can we do a `visibility: hidden;` to make the intent of the div a little more clear?